AK/BufferStream.h

```diff
@@ -126,6 +187,12 @@ class BufferStream {
         m_buffer.trim(m_offset);
     }
 
+    void reset()
```

Comment:
Oh, I suppose this should be a separate commit. And I should probably fix the = 0 -> = false :)